druid/druid.sh

```diff
@@ -0,0 +1,320 @@
+[Unit]
+Description=Overlord
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat conf/druid/overlord/jvm.config | xargs` -cp "${DRUID_DIR}/conf/druid/_common:${DRUID_DIR}/conf/druid/_common/hadoop-xml:${DRUID_DIR}/conf/druid/overlord:lib/*" io.druid.cli.Main server overlord"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/druid-broker.service
+[Unit]
+Description=Broker
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat examples/conf/druid/broker/jvm.config | xargs` -cp "${DRUID_DIR}/examples/conf/druid/_common:${DRUID_DIR}/examples/conf/druid/_common/hadoop-xml:${DRUID_DIR}/examples/conf/druid/broker:lib/*" io.druid.cli.Main server broker"
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/druid-historical.service
+[Unit]
+Description=Historical
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat conf/druid/historical/jvm.config | xargs` -cp "${DRUID_DIR}/conf/druid/_common:${DRUID_DIR}/conf/druid/_common/hadoop-xml:${DRUID_DIR}/conf/druid/historical:lib/*" io.druid.cli.Main server historical"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  cat << EOF > /etc/systemd/system/druid-middle-manager.service
+[Unit]
+Description=MiddleManager
+Wants=network-online.target
+After=network-online.target
+[Service]
+WorkingDirectory=${DRUID_DIR}
+Environment="JAVA_HOME=${java_home}"
+Environment="HADOOP_CONF_DIR=${hadoop_conf_dir}"
+
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash -c "java `cat examples/conf/druid/middleManager/jvm.config | xargs` -cp "${DRUID_DIR}/examples/conf/druid/_common:${DRUID_DIR}/examples/conf/druid/_common/hadoop-xml:${DRUID_DIR}/examples/conf/druid/middleManager:lib/*" io.druid.cli.Main server middleManager"
+[Install]
+WantedBy=multi-user.target
+EOF
+}
+
+function start_druid(){
+  cd ${DRUID_DIR}
+  systemctl daemon-reload
+  systemctl enable druid-coordinator druid-overlord druid-broker druid-historical druid-middle-manager
+  systemctl start druid-coordinator druid-overlord druid-broker druid-middle-manager druid-historical
+}
+
+function main() {
+  readonly enable_cloud_sql_metastore="$(/usr/share/google/get_metadata_value \
+    attributes/enable-cloud-sql-hive-metastore || echo 'true')"
+  local role=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+  if [[ ${role} == 'Master' ]]; then
```

Comment:
We need to setup druid in following way:

Master:
Coordinator Nodes
Broker Nodes
Overlord Nodes

Workers:
Historical Nodes
Realtime Nodes
MiddleManager Nodes